docs/deployment-examples/sql-server-databases.md

```diff
@@ -1,48 +1,91 @@
 ---
 title: SQL Server Databases
-description: Options for deploying to SQL Server databases including a model-driven approach and a change-script-driven approach.
+description: Octopus integrates with numerous tools to handle automated database deployments.
 position: 30
 ---
 
-There are a variety of ways that SQL Server databases can be managed in Octopus Deploy. This section will provide a broad outline of the different approaches, and then drill into one possible solution to use as an example.
-
-:::hint
-**Tentacle Installation Location**
-For SQL deployments, the SQL server does not require a locally installed Tentacle. You can use any Tentacle installed on any server, so long as it has access to SQL to run the functions you require.
-:::
+There are a variety of ways for Octopus Deploy to deploy to SQL Server.  There are many third-party tools, both free and commercial, Octopus Deploy integrates with.  This section will provide a broad outline of approaches, tooling, and recommended configuration.
 
 ## Approaches to database change management {#SQLServerdatabases-Approachestodatabasechangemanagement}
 
-There are two main approaches to managing database definitions and applying differences when deploying:
+There are two main approaches to deploying databases.
+  
+1. A model-driven approach, where you define the desired state of your database.  During deployment, a tool will compare the desired state with the target database.  Using that comparison it will generate a delta script.
+
+[Redgate's DLM Automation Suite](https://www.red-gate.com/products/dlm/dlm-automation/index), [Microsoft's DacPac](https://docs.microsoft.com/en-us/sql/relational-databases/data-tier-applications/data-tier-applications?view=sql-server-2017), and [Microsoft's Entity Framework Migrations](https://msdn.microsoft.com/en-us/library/jj591621(v=vs.113).aspx) use the model-driven approach.
+
+2. A change-driven approach, where schema changes are manually written and only run once.  The target database keeps track of which scripts already ran.
+
+[Redgate's ReadyRoll](https://www.red-gate.com/products/sql-development/readyroll/index), [DbUp](https://dbup.readthedocs.io/en/latest/), and [RoundhousE](http://projectroundhouse.org/) use the change-driven approach.
+
+There are pros and cons to either approach as well as the tools themselves.  It is important for your team to research the tools and determine the best one for you.
+
+## Tentacle Installation Recommendations {#SQLServerdatabases-Tentacles}
+
+Deploying an IIS Web Application or a Windows Service is very straight-forward.  Install the tentacle on the server to be deployed to.  SQL Server is different.  Production SQL Servers are typically clusters or high-availability groups.  They comprise more than one node hidden behind a VIP or virtual IP Address. 
```

Comment:
Tentacle should be capitalized throughout.

Reply:
Thank you, fixed.